resource/opentelemetry-resource-detector-container/README.rst

```diff
@@ -0,0 +1,47 @@
+OpenTelemetry Resource detectors for containers
+==========================================================
+
+|pypi|
+
+.. |pypi| image:: TODO
+   :target: TODO
+
+
+This library provides container property (container.id) detection features which can help
+in identifying the problems in app - infra correlation for apps running on containers i.e. docker
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-resource-detector-containerß
+
+---------------------------
+
+Below is the give example for `opentelemetry-resource-detector-kubernetes`
```

Comment:
Why is kubernetes specifically mentioned here? Docker is mentioned elsewhere. Is this approach universal to any container manager? If so, we should avoid specific names.